test/cases/rhel_8-s390x-tar-boot.json

```diff
@@ -2196,6 +2203,10 @@
             }
           }
         },
+        {
+          "name": "org.osbuild.zipl",
```

Comment:
@gicmo , @teg : Do we want the zipl stage in `tar` image type or not?

Reply:
The tar output type is under-specified, so for now I think we should just leave it as is. I would like to revisit this, and get proper specifications.